ui/admin/tests/acceptance/targets/credential-sources-test.js

```diff
@@ -104,13 +119,51 @@ module('Acceptance | targets | credential-libraries', function (hooks) {
     assert.strictEqual(currentURL(), urls.addCredentialSources);
   });
 
+  test('displays list of credential sources with only credential libraries available', async function (assert) {
```

Comment:
Is there a test for when there's both credential libraries and credentials available?

Reply:
🤦‍♂️ I was too focused on edge cases I forgot to test the normal behavior. Adding now.